f3_test.go

```diff
@@ -484,10 +547,10 @@ func (e *testEnv) waitFor(f func(n *testNode) bool, timeout time.Duration) {
 }
 
 func (e *testEnv) waitForNodesInitialization() {
-	f := func(n *testNode) bool {
+	e.t.Helper()
+	e.waitFor(func(n *testNode) bool {
 		return n.f3.IsRunning()
-	}
-	e.waitFor(f, 5*time.Second)
+	}, 30*time.Second)
```

Comment:
This probably isn't necessary anymore (I was trying to fix an issue where the race-detector slowed things down too much) but... 5s was unnecessarily short anyways.